src/cli/README.md

```diff
@@ -1169,6 +1169,25 @@ DNS response for example.com. - fd4c:9574:3720:2:0:0:5db8:d822 TTL:20456
 Done
 ```
 
+### dns resolve4 \<hostname\> \[DNS server IP\] \[DNS server port\] \[response timeout (ms)\] \[max tx attempts\] \[recursion desired (boolean)\]
+
+Send DNS query to obtain IPv4 address for a given hostname.
```

Comment:
The output doesn't give us the IPv4 address though, so should we revise this to be more in line with what it actually does? (basically the statement on line 1182)

Reply:
Moved the sentence up.